### ISSUE TITLE
Module build failed (from ...../node_modules/mini-css-extract-plugin/dist/loader.js)

### ISSUE THREAD

**Issue by Reichbaum** [WebDevStudios/wd_s#760]
Hi! I'm having trouble installing the latest theme build. All the time the same error during the build - with mini-css-extract-plugin. I've tried on different versions of node - 14.18.1, 15.14.0, 16.11.1 - always the same result. Can you help me?

[2021-10-13T17_15_24_032Z-debug.log](https://github.com/WebDevStudios/wd_s/files/7340193/2021-10-13T17_15_24_032Z-debug.log)

![image](https://user-images.githubusercontent.com/52535392/137182660-bf435d20-8197-4537-aa93-36d431b75cdb.png)

**Comment by coreymcollins** [comment]
Are you running on Windows? It looks like you're running npm version 8 as well, can you confirm that?

I'm not on Windows but we do have a couple of engineers here who are, so I can see if this is anything they've run into at any point.

**Comment by coreymcollins** [comment]
Hey @Reichbaum – one of our Windows folks let me know that they ran into a similar issue and had to run `npm install -g npm` to solve it. Can you run that and then try running the setup steps again to see if it solves the issue for you as well?

Thanks!

**Comment by Reichbaum** [comment]
Hey! Yes, I am on Windows 10. I did `npm install -g npm` to version 8.1.0, but that didn't help. The result is the same. 
[2021-10-15T06_44_30_054Z-debug.log](https://github.com/WebDevStudios/wd_s/files/7351428/2021-10-15T06_44_30_054Z-debug.log)

**Comment by coreymcollins** [comment]
Not being on Windows it's hard for me to dig super deep into this, but I'm curious if maybe composer isn't installed on the same drive as your theme? I see this in your debug log:
```
9696 error path D:\#Websites\christmas-music\app\public\wp-content\themes\christmas-music
9697 error command failed
9698 error command C:\Windows\system32\cmd.exe /d /s /c composer install --quiet && npm run build
```

And to me, it looks like your theme is installed on the D drive but the error may be coming from the C drive? (doing a quick search, there are folks who have had a similar issue: https://github.com/composer/composer/issues/8680 and https://stackoverflow.com/questions/24664269/install-composer-for-windows-on-d-drive).

I also wonder if the `#` in your path causes some kind of parsing issue. Have you used the theme without issue on the same drive with the same `\#Websites\` path before?

Thanks!

**Comment by Reichbaum** [comment]
Corey, you are a wonder! 💪

However, I can't determine with 100% certainty what the problem was, because I didn't keep the experiment clean. 
I uninstalled Composer and installed it on the D. Then, I prescribed a global path to it. It responded in the terminal, however, it still gave the same familiar errors when installing the theme. Then I decided to reboot the computer and that may have caused blessed Windows to really see the path to Composer. But additionally, I decided to test the "#" symbol hypothesis and created a new site with the theme in a folder without it (by the way, the old builds worked fine in that folder with "#"). 
And it worked! All in all, some of that, or collectively, helped. 

I appreciate you so much!
